setup.py

```diff
@@ -42,6 +42,7 @@
         "hashids>=1.3,<1.4",
         "cryptography>=3.3.1,<4.0",
         "tldextract>=2.2,<2.3",
+        "masonite-validation>=4.0b1",
```

Comment:
So is it needed finally ? If all masonite-validation code has been integrated into core ?

Reply:
Good point. I'll remove this